### DISCUSSION TITLE
Any way to migrate/import blocked numbers from QKSMS?

### DISCUSSION THREAD

**Discussion by studioph** [octoshrimpy/quik#164] [Q&A]
I have a ton of blocked numbers on QKSMS espcially due to all of the election spam happening. I recently discovered Quik and installed it and while it migrated all the converstaions fine, that also includes *all* of the previously blocked messages and none of the blocked numbers got carried over. I *really* hate the idea of having to start over from scratch...is there really no way to move these over?

**Comment by octoshrimpy** [comment]
I don't believe QKSMS has a way of exporting blocked numbers. Due to that, I can't think of a clean solution to the problem right now. any suggestion is welcome :)
